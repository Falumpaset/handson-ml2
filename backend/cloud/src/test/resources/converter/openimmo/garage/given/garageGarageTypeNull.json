{
  "aktion": "CHANGE",
  "supplierNumber": "test",
  "applicationLink": "http://www.test.de",
  "shortDescription": "Test Objektbeschreibung",
  "defaultContactEmail": "test@immomio.de",
  "countryCode": "DEU",
  "now": "2020-06-16T00:00:00Z",
  "property": {
    "id": 1,
    "customer": {
      "name": "Fabian Beck Immobilien"
    },
    "user": {
      "profile": {
        "firstname": "Fabian",
        "name": "Beck"
      }
    },
    "externalId": 1,
    "data": {
      "name": "Freie Garage in Hamburg in der Elmenhorststraße 1, 22767 Hamburg, DE",
      "size": 50.0,
      "attic": false,
      "floor": 3,
      "rooms": null,
      "garden": false,
      "ground": null,
      "heater": null,
      "address": {
        "city": "Hamburg",
        "region": "Hamburg",
        "street": "Elmenhorststraße",
        "country": "DE",
        "zipCode": "22767",
        "additional": null,
        "coordinates": null,
        "houseNumber": "1"
      },
      "contact": null,
      "seniors": false,
      "bailment": 2000.0,
      "elevator": false,
      "flatType": null,
      "landArea": null,
      "basePrice": 1000.0,
      "bathRooms": null,
      "documents": [],
      "fireplace": false,
      "flatShare": false,
      "gardenUse": false,
      "houseType": null,
      "storeRoom": false,
      "externalId": null,
      "garageData": {
        "width": null,
        "height": null,
        "length": null,
        "freeUntil": 1592438400000,
        "garageType": null
      },
      "gardenArea": null,
      "objectType": "GARAGE",
      "tvSatCable": false,
      "attachments": [],
      "barrierFree": false,
      "bicycleRoom": false,
      "heatingCost": null,
      "kitchenette": false,
      "referenceId": 1,
      "showAddress": true,
      "showContact": false,
      "washDryRoom": false,
      "basementSize": null,
      "availableFrom": {
        "dateAvailable" : "2020-06-17",
        "stringAvailable" : null
      },
      "guestToilette": false,
      "parkingSpaces": null,
      "serviceCharge": null,
      "commercialData": null,
      "furnishingType": null,
      "numberOfFloors": 3,
      "previousTenant": null,
      "totalRentGross": null,
      "temporaryLiving": null,
      "constructionYear": "2000",
      "historicBuilding": false,
      "numberOfBedrooms": null,
      "numberOfTerraces": null,
      "basementAvailable": null,
      "bathroomEquipment": null,
      "buildingCondition": null,
      "energyCertificate": null,
      "lastRefurbishment": null,
      "numberOfBalconies": null,
      "objectDescription": "Test Objektbeschreibung",
      "totalParkingPrice": null,
      "balconyTerraceArea": null,
      "objectLocationText": "Test Lage",
      "heatingCostIncluded": false,
      "furnishingDescription": "Test Ausstatt",
      "objectMiscellaneousText": "Test sonstige Angaben",
      "parkingPriceIncludedToAdditionalCosts": false
    }
  }
}