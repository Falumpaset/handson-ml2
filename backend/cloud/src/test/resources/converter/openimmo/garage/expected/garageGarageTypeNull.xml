<?xml version="1.0" ?>
<openimmo xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="">
    <uebertragung art="ONLINE" umfang="TEIL" version="1.2.7" sendersoftware="Immomio"
                  senderversion="1.0"></uebertragung>
    <anbieter>
        <anbieternr>jbbg402699</anbieternr>
        <firma>test</firma>
        <openimmo_anid>AIMIO16159437268522153127c0b5da</openimmo_anid>
        <immobilie>
            <objektkategorie>
                <nutzungsart WOHNEN="false" GEWERBE="false" ANLAGE="true"></nutzungsart>
                <vermarktungsart KAUF="false" MIETE_PACHT="true"></vermarktungsart>
                <objektart>
                    <parken parken_typ="STELLPLATZ"></parken>
                </objektart>
            </objektkategorie>
            <geo>
                <plz>22767</plz>
                <ort>Hamburg</ort>
                <strasse>Elmenhorststraße</strasse>
                <hausnummer>1</hausnummer>
                <bundesland>Hamburg</bundesland>
                <land iso_land="DEU"></land>
                <etage>3</etage>
                <anzahl_etagen>3</anzahl_etagen>
            </geo>
            <kontaktperson>
                <email_zentrale>test@immomio.de</email_zentrale>
                <email_direkt>test@immomio.de</email_direkt>
                <name>Beck</name>
                <vorname>Fabian</vorname>
                <firma>Fabian Beck Immobilien</firma>
            </kontaktperson>
            <preise>
                <kaltmiete>1000.00</kaltmiete>
                <gesamtmietebrutto>1000.00</gesamtmietebrutto>
                <kaution>2000.00</kaution>
            </preise>
            <flaechen>
                <nutzflaeche>50.0</nutzflaeche>
            </flaechen>
            <zustand_angaben>
                <baujahr>2000</baujahr>
            </zustand_angaben>
            <freitexte>
                <objekttitel>Freie Garage in Hamburg in der Elmenhorststraße 1, 22767 Hamburg, DE</objekttitel>
                <lage>Test Lage</lage>
                <ausstatt_beschr>Test Ausstatt</ausstatt_beschr>
                <objektbeschreibung>Test Objektbeschreibung</objektbeschreibung>
                <sonstige_angaben>Test sonstige Angaben</sonstige_angaben>
            </freitexte>
            <anhaenge>
                <anhang location="REMOTE" gruppe="LINKS">
                    <anhangtitel>Interessentenlink</anhangtitel>
                    <daten>
                        <pfad>http://www.test.de</pfad>
                    </daten>
                </anhang>
            </anhaenge>
            <verwaltung_objekt>
                <objektadresse_freigeben>true</objektadresse_freigeben>
                <verfuegbar_ab>2020-06-17</verfuegbar_ab>
                <bisdatum>2020-06-18T00:00:00Z</bisdatum>
            </verwaltung_objekt>
            <verwaltung_techn>
                <objektnr_intern>1</objektnr_intern>
                <objektnr_extern>1</objektnr_extern>
                <aktion aktionart="CHANGE"></aktion>
                <openimmo_obid>1</openimmo_obid>
                <stand_vom>2020-06-16T00:00:00Z</stand_vom>
            </verwaltung_techn>
        </immobilie>
    </anbieter>
</openimmo>